todo_app/helpers/list_id_helper.py

```diff
@@ -0,0 +1,7 @@
+import os
+list_id_to_do = os.getenv('TO_DO_LIST_ID') 
+list_id_done = os.getenv('DONE_LIST_ID')
+
+LIST_ID ={
+    list_id_to_do: "To do",
+    list_id_done: "Done"}
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it. 

The reason it could be a problem is that these lines of code run when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a solution might be to just load the environment variables when they're needed, e.g. something like:
```python
def get_list_name(id):
    list_id_to_do = os.getenv('TO_DO_LIST_ID') 
    list_id_done = os.getenv('DONE_LIST_ID')
    LIST_ID ={
        list_id_to_do: "To do",
        list_id_done: "Done" 
    }
    return LIST_ID[id]

# or indeed:
def get_list_id_map():
    list_id_to_do = os.getenv('TO_DO_LIST_ID') 
    list_id_done = os.getenv('DONE_LIST_ID')

    LIST_ID ={
        list_id_to_do: "To do",
        list_id_done: "Done"
    }
    return LIST_ID
```